lib/katello/plugin.rb

```diff
@@ -271,19 +271,21 @@
       common_td_class = "#{common_th_class} ellipsis"
       use_pagelet :hosts_table_column_header, :name
       use_pagelet :hosts_table_column_content, :name
-      add_pagelet :hosts_table_column_header, key: :subscription_status, label: _('Subscription status'), sortable: true, class: common_th_class, width: '10%'
+      add_pagelet :hosts_table_column_header, key: :subscription_status, label: _('Subscription status'), sortable: true, class: common_th_class, width: '10%', export_key: 'subscription_global_status'
       add_pagelet :hosts_table_column_content, key: :subscription_status, class: common_td_class, callback: ->(host) { host_status_icon(host.subscription_global_status) }
-      add_pagelet :hosts_table_column_header, key: :installable_updates, label: _('Installable updates'), class: common_th_class, width: '15%'
+      add_pagelet :hosts_table_column_header, key: :installable_updates, label: _('Installable updates'), class: common_th_class, width: '15%',
+                  export_data: [:security, :bugfix, :enhancement].map { |kind| CsvExporter::ExportDefinition.new("installable_updates.#{kind}", callback: ->(host) { (host.content_facet_attributes&.errata_counts || {})[kind] }) } +
+                               [:rpm, :deb].map { |kind| CsvExporter::ExportDefinition.new("installable_packages.#{kind}", callback: ->(host) { host&.content_facet_attributes&.public_send("upgradable_#{kind}_count".to_sym) || 0 }) }
       add_pagelet :hosts_table_column_content, key: :installable_updates, class: common_td_class, callback: ->(host) { errata_counts(host) }
       use_pagelet :hosts_table_column_header, :os_title
       use_pagelet :hosts_table_column_content, :os_title
-      add_pagelet :hosts_table_column_header, key: :lifecycle_environment, label: _('Lifecycle environment'), sortable: true, class: common_th_class, width: '10%'
+      add_pagelet :hosts_table_column_header, key: :lifecycle_environment, label: _('Lifecycle environment'), sortable: true, class: common_th_class, width: '10%', export_key: 'single_lifecycle_environment'
       add_pagelet :hosts_table_column_content, key: :lifecycle_environment, class: common_td_class, callback: ->(host) { host.content_facet&.single_lifecycle_environment&.name }
-      add_pagelet :hosts_table_column_header, key: :content_view, label: _('Content view'), sortable: true, class: common_th_class, width: '10%'
+      add_pagelet :hosts_table_column_header, key: :content_view, label: _('Content view'), sortable: true, class: common_th_class, width: '10%', export_key: 'single_content_view'
```

Comment:
These headers seem to be picking up label based on key instead of the actual label param.
![Screenshot from 2023-03-01 16-21-47](https://user-images.githubusercontent.com/21146741/222267727-36a319f0-3822-4616-94c2-0ad95d898ff0.png)

Reply:
Yes, that's the way it works. The label itself is translated which is not really desirable in csv exports. I mean, there are ways to force it to be named differently if that's what you want

Reply:
That would be ideal. 👍🏼